### PR TITLE
TT-305 improve tib docs

### PR DESCRIPTION
**Pull request by sredxny**
PR for ticket: https://tyktech.atlassian.net/browse/TT-305

Missing things:
- Tykify diagrams (Someone else in tyk will do so...who?)
- Once Paul upload the SAML videos to youtube, we need to embeded them in the docs as well

### CODE REVIEW

**Comment by sredxny** [comment]
tib-ui

File: tyk-docs/content/advanced-configuration/integrate/sso/dashboard-login-azure-sso.md [hunk 1]
```diff
@@ -87,6 +87,7 @@ curl http://{TIB-DOMAIN}:{TIB-PORT}/api/profiles/{PROFILE-NAME-IN-TIB} -H "Autho
  5. TIB redirected the call to the dashboard to a special endpoint `/tap` ( it was defined on the profile under `ReturnURL`) with the nonce that was created.
  6. The Dashboard on the `/tap` endpoint finds the session that is attached to the `nonce`, login the user and redirect to the dashboard first page
 
+![Generate Or Login User Profile flow](/docs/img/diagrams/generate-or-login-user-profile.png)
```
Comment (letzya) [inline tyk-docs/content/advanced-configuration/integrate/sso/dashboard-login-azure-sso.md:90]: @sredxny do you think we should add the logic we have for the user lookup? if user not found - create an ad hoc user session in Redis and if found in mongo we use that user’s object.

**Review by marksou — APPROVED** [review]
Looks Good

**Review by gregdelhon — APPROVED** [review]
LGTM

**Pull request merged**